src/averageAges.js

```diff
@@ -20,6 +20,18 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  let onlyMen;
+
+  arguments.length === 1
+    ? onlyMen = people.filter(person => person.sex === 'm')
+    : onlyMen = people.filter(person => person.sex === 'm'
+    && Math.ceil(person.died / 100) === century);
```

Comment:
```suggestion
  const men = people.filter(person => person.sex === 'm');
  const onlyMen = century
    ? men.filter(person => Math.ceil(person.died / 100) === century)
    : men;
```

В остальных функциях аналогично